ui/imports/utils/Constants.qml

```diff
@@ -958,7 +958,8 @@ QtObject {
     enum TokenType {
         Unknown = 0,
         ERC20 = 1, // Asset
-        ERC721 = 2 // Collectible
+        ERC721 = 2, // Collectible
+        ENS = 3
     }
```

Comment:
Not really a token, is it? Could we use another enum instead. Something specific to Perimissions?

Reply:
Maybe not, dunno, I just added here merely for completeness, it was missing compared to the backend